lib/src/system_cache.dart

```diff
@@ -89,7 +89,7 @@ class SystemCache {
   /// Defaults to `false`.
   SystemCache({String rootDir, bool isOffline = false})
       : rootDir = rootDir ?? SystemCache.defaultDir,
-        tokenStore = TokenStore(rootDir ?? SystemCache.defaultDir) {
+        tokenStore = TokenStore(dartConfigDir) {
```

Comment:
Let's also rename `tokens.json` to `pub-tokens.json`.

Reply:
Yes!
Done!